<launch>
  <node pkg="tf2_ros" type="static_transform_publisher" name="ros_tf_static_map_origin" args="0 0 0 0 0 0 origin map" />
  <rosparam command="load" file="$(find path_plan)/conf/waypoints_list.yml" />
  <param name="acceptable_deviation" type="double" value="0.3" />
  <param name="start_pose_x" type="double" value="5" />
  <param name="start_pose_y" type="double" value="5" />
  <param name="start_pose_z" type="double" value="5" />
  <node name="path_planner_node" pkg="path_plan" type="path_planing.py" output="screen"/>

</launch>